packages/platform/types/event-types/event-types_2024_06_14/inputs/create-event-type.input.ts

```diff
@@ -106,9 +112,50 @@ export class CreateEventTypeInput_2024_06_14 {
   offsetStart?: number;
 
   @IsOptional()
+  @Type(() => BookerLayouts_2024_06_14)
+  bookerLayouts?: BookerLayouts_2024_06_14;
+
+  @IsOptional()
+  @ValidateRequiresConfirmation()
+  @Type(() => RequiresConfirmation_2024_06_14)
+  requiresConfirmation?: RequiresConfirmation_2024_06_14;
+
   @ValidateNested()
   @Type(() => Recurrence_2024_06_14)
   recurrence?: Recurrence_2024_06_14;
+
+  @IsOptional()
+  @IsBoolean()
+  requiresBookerEmailVerification?: boolean;
+
+  @IsOptional()
+  @IsBoolean()
+  hideCalendarNotes?: boolean;
+
+  @IsOptional()
+  @IsBoolean()
+  lockTimeZoneToggleOnBookingPage?: boolean;
+
+  @IsOptional()
+  @Type(() => EventTypeColor_2024_06_14)
+  eventTypeColor?: EventTypeColor_2024_06_14;
```

Comment:
Since we already are in create event type input, i think we can rename `eventTypeColor` to simply `color`

Reply:
fixed - https://github.com/calcom/cal.com/commit/c58c19e6f992f52689535722045f6420ce823a79